topi/tests/python/common.py

```diff
@@ -9,4 +9,4 @@ def get_all_backend():
         A list of all supported targets
     """
     return ['llvm', 'cuda', 'opencl', 'metal', 'rocm', 'vulkan', 'nvptx',
-            'llvm -device=arm_cpu']
+            'llvm -device=arm_cpu', 'aocl_sw_emu']
```

Comment:
It seems like we are now testing AOCL backend for all unit tests. What are the pre-requisites to allow for AOCL emulation? I assume that the user will need to have the toolchains installed on their machines. What if they don't -- will it break the unit tests? Are those toolchains easy to install in a docker image, for CI testing?

Reply:
Thanks for confirming that the test will be skipped if the Intel SDK is not installed.